command/harness/delegate/delegate.go

```diff
@@ -70,6 +70,9 @@ func (c *delegateCommand) run(*kingpin.ParseContext) error {
 	if err != nil {
 		return err
 	}
+	if env.Settings.HarnessDownloadURI == "" {
```

Comment:
not needed if you set the default

Reply:
I explicitly didn't set default as we only want this value to be set for delegate and dlite. If I'll set a default, it'll be present for other runners as well which we don't want